opm/parser/eclipse/EclipseState/Tables/ImptvdTable.hpp

```diff
@@ -20,100 +20,77 @@
 #define	OPM_PARSER_IMPTVD_TABLE_HPP
```

Comment:
ok, the weird white space was already there before. I think it would be good to fix all of them in one go, though.